src/main/java/net/minecraftforge/fml/common/registry/PersistentRegistryManager.java

```diff
@@ -476,6 +496,7 @@ else if (action == FMLMissingMappingsEvent.Action.BLOCKONLY)
                 if (action == FMLMissingMappingsEvent.Action.DEFAULT)
                 {
                     defaulted.add(remap.name);
+                    defaultedShort.add(remap.name);
                 }
```

Comment:
defaultedShort gives users a count of the *names* which were removed - the default count doesn't take into account that, for instance, blocks and items with the same ID are usually the same thing. It's for user friendliness.